app/api.py

```diff
@@ -208,6 +208,17 @@ def get_user_proofs(
     return crud.get_user_proofs(db, user=current_user)
 
 
+@app.get("/products/{product_id}", response_model=schemas.ProductBase)
+async def get_product(product_id: int, db: Session = Depends(get_db)):
```

Comment:
We should (almost) always declare our function as normal function (def myfunc), and not coroutine (async def), as we don't support async currently:
see https://fastapi.tiangolo.com/async/

Reply:
i removed async everywhere :ok_hand: